src/components/shared/MarkerTooltipContents.js

```diff
@@ -12,10 +12,13 @@ import {
   formatBytes,
   formatValueTotal,
 } from '../../utils/format-numbers';
+import Backtrace from '../header/Backtrace';
```

Comment:
maybe `Backtrace` should be in `shared` ? I think that nothing in `shared` should depend on anything else than `shared` (except maybe `utils`); moreover it's not used in the header only, but also in the marker view.